// star-labs/data.js
module.exports = {
    interact(key){
      let result;
      let hug = [""];
      let kill = [""];
      let lick = [""];
      let kiss = ["https://i.imgur.com/zSMRiE1.gif",
                  "https://i.imgur.com/o2WurZT.mp4",
                  "https://i.imgur.com/b9x8hqa.gif",
                  "https://i.imgur.com/nVsyLqY.gif",
                  "https://i.imgur.com/LnRErNi.mp4",
                  "https://i.imgur.com/0UL9ehz.gif",
                  "https://i.imgur.com/jyR9zrL.gif",
                  "https://i.imgur.com/I00ZyAk.gif",
                  "https://i.imgur.com/qZNavF4.gif",
                  "https://i.imgur.com/aeDFgUM.gif",
                  "https://i.imgur.com/JObPEp1.mp4",
                  "https://i.imgur.com/ZvCVcO7.gif",
                  "https://i.imgur.com/R320us5.gif",
                  "https://i.imgur.com/4KjnlAe.gif",
                  "https://i.imgur.com/tPxq4bY.gif"];
      let feed = [""];
      let slap = [""];
      let kick = [""];
      let poke = [""];
      let cuddle = [""];
      let highfive = [""];
      let punch = [""];
      let tickle = [""];
      let bite = [""];
      let handholding = [""];
      let baka = [""];
      let kickbutts = [""];
      let splash = [""];
      let claps = [""];
      let cheeks = [""];
      let scared = [""];
      let laugh = [""];
      let hello = [""];
      switch(key){
        case 'kiss':
          key = Math.floor((Math.random() * kiss.length))
          result = kiss[key];
          return result;
          break;
      }
    },
    reaction(key){
      let result;
      let suicide = [""];
      let happy = [""];
      let blush = [""];
      let dance = [""];
      let confused = [""];
      let cry = [""];
      let like = [""];
      let boom = [""];
      let sleep = [""];
      let facepalm = [""];
      let pervert = [""];
      let dab = [""];
      let angry = [""];
      let banghead = [""];
      let no = [""];
      let pout = [""];
      let shrug = [""];
      let smug = [""];
      let bored = [""];
      let disgust = [""];
      let vomit = [""];
      let fbi = [""];
      let run = [""];
      let think = [""];
    },
    nsfw(key){
      let result;
      let fuck = [""];
      let tits = [""];
      let ass = [""];
      let rahentai = [""];
      let hnekogirl = [""];
      let konachan = [""];
      let gmasturbate = [""];
      let suck = [""];
      let cum = [""];
      let cumMention = [""];
      let blowjob = [""];
    },
    suicide() {
      const links = ["https://media.giphy.com/media/Qx4CJBlMgEBoUH3p2c/giphy.gif", 
                     "https://media.giphy.com/media/SqrZQ921epXZbXsIeP/giphy.gif", 
                     "https://media.giphy.com/media/ghCVdQeMBgq1Fjznqk/giphy.gif", 
                     "https://media.giphy.com/media/RGXKhAAt9CcnwbHxiu/giphy.gif", 
                     "https://media.giphy.com/media/fAo0AIAxgyurMR1VdR/giphy.gif", 
                     "https://media.giphy.com/media/Ky5IGNQoJH2AIlbkyF/giphy.gif", 
                     "https://media.giphy.com/media/YnY2jaK8xqaGtSKKO6/giphy.gif", 
                     "https://media.giphy.com/media/JQRWuqMBJq4hYEZrsS/giphy.gif", 
                     "https://media.giphy.com/media/Ur71tO5howy515b6Mz/giphy.gif"];
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    hug(){
      const links = ["https://media.giphy.com/media/gGqbXTa8TZn2gWlfk8/giphy.gif", 
                     "https://media.giphy.com/media/elOxEBAaw0yPGcypBL/giphy.gif", 
                     "https://media.giphy.com/media/ZEBPtMdWHcDikGn4Pi/giphy.gif", 
                     "https://media.giphy.com/media/RIwS1ghBmj2H3J9PzN/giphy.gif", 
                     "https://media.giphy.com/media/jtu0BtJQKsqaOpsfMf/giphy.gif", 
                     "https://media.giphy.com/media/S3hULwj906j4x9DvPR/giphy.gif"];
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    kill(){
      const links = ["https://media.giphy.com/media/rDOW7YUXV4IOA/giphy.gif", 
                     "https://media.giphy.com/media/eYAYL9QMjZyE0/giphy.gif", 
                     "https://media.giphy.com/media/jwuswmKDBsVJC/giphy.gif", 
                     "https://media.giphy.com/media/ZWjz534ezT1YY/giphy.gif"];
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    lick(){
      const links = ["https://media.giphy.com/media/WQIViBW0KrBlpEQe8S/giphy.gif", 
                     "https://media.giphy.com/media/JtM7CrUUCkwked1XtL/giphy.gif", 
                     "https://media.giphy.com/media/iG5AeD2gFPOvof8kcF/giphy.gif", 
                     "https://media.giphy.com/media/eKgNfBzHkf5af6Ho8R/giphy.gif", 
                     "https://media.giphy.com/media/LPU0KiyQ2s8DDnhlZH/giphy.gif", 
                     "https://media.giphy.com/media/KDVrHbNmVMl8ZPSgoJ/giphy.gif", 
                     "https://media.giphy.com/media/WrOv3EYglC0rZaWORj/giphy.gif", 
                     "https://media.giphy.com/media/f4aQynCL18XQSrwUkH/giphy.gif"];
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    slap(){
      const links = ["https://media.giphy.com/media/TJZg06g4gtUbaWqoYU/giphy.gif", 
                     "https://media.giphy.com/media/VDjzAy10WtxMMgJMUq/giphy.gif", 
                     "https://media.giphy.com/media/ehO1XO8jaHpl6z7qcy/giphy.gif", 
                     "https://media.giphy.com/media/XfaLLmzlXbymdcJhbR/giphy.gif", 
                     "https://media.giphy.com/media/UtiKrExwsOVIUkQCBe/giphy.gif", 
                     "https://media.giphy.com/media/WSrMYj34VcW7ReylR8/giphy.gif", 
                     "https://media.giphy.com/media/IeS269JgrMvduPm4gR/giphy.gif", 
                     "https://media.giphy.com/media/gFuP3hSan8stIojAio/giphy.gif"];
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    happy(){
      const links = ["https://media.giphy.com/media/fVt9ofo9kOj0KvMmNV/giphy.gif", 
                     "https://media.giphy.com/media/YRJdpNvjRwHi9rgLDX/giphy.gif", 
                     "https://media.giphy.com/media/KeQcRsDateZWAxh3oF/giphy.gif", 
                     "https://media.giphy.com/media/TLnGOtSjpGKsfiB9o3/giphy.gif", 
                     "https://media.giphy.com/media/RhkdMymXomkpCBE0d6/giphy.gif", 
                     "https://media.giphy.com/media/Xevj6SReFwfPaewPXO/giphy.gif", 
                     "https://media.giphy.com/media/ejDlpvSXeVSJMNRJgV/giphy.gif", 
                     "https://media.giphy.com/media/XgH3oqDECKXemAn3dx/giphy.gif", 
                     "https://media.giphy.com/media/l3mHWBusS0WECVsXqy/giphy.gif"];
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    cry(){
      const links = ["https://media.giphy.com/media/3fmRTfVIKMRiM/giphy.gif", 
                     "https://media.giphy.com/media/XmPrx6vcB0X6g/giphy.gif", 
                     "https://media.giphy.com/media/4NuAILyDbmD16/giphy.gif", 
                     "https://media.giphy.com/media/eHekyNso61EqY/giphy.gif", 
                     "https://media.giphy.com/media/OPU6wzx8JrHna/giphy.gif", 
                     "https://media.giphy.com/media/ROF8OQvDmxytW/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    kiss(){
      const links = ["https://media.giphy.com/media/iGve29H7rFI0YqzMAO/giphy.gif", 
                     "https://media.giphy.com/media/KAdcKW4YLXylKbnke3/giphy.gif", 
                     "https://media.giphy.com/media/gEvg4S2B6OKvxvGf74/giphy.gif", 
                     "https://media.giphy.com/media/Z9hI8OIAL6pZ2ie6tJ/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    feed(){
      const links = ["https://media.giphy.com/media/gFEj9y3Tw8GfDlkGDr/giphy.gif", 
                     "https://media.giphy.com/media/Y3YBQn3YqyIBuF3qcq/giphy.gif", 
                     "https://media.giphy.com/media/hsmrpdtUzwHTo0wjxN/giphy.gif", 
                     "https://media.giphy.com/media/ka59O7lidZvPQhLkAg/giphy.gif", 
                     "https://media.giphy.com/media/LOXHvNVHgyQJ5DpwOQ/giphy.gif", 
                     "https://media.giphy.com/media/LkTFbVZE8BY93Yuv7p/giphy.gif", 
                     "https://media.giphy.com/media/Md9Zx3tpUN0L3BANOi/giphy.gif", 
                     "https://media.giphy.com/media/RlTfgCb02Bxvflva9d/giphy.gif", 
                     "https://media.giphy.com/media/Tk1LqJsunDLvBBifrA/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    pat(){
      const links = ["https://media.giphy.com/media/j6ZRb6vhU0fx3i7CUo/giphy.gif", 
                     "https://media.giphy.com/media/TGWmCFP1FBuwE4tawm/giphy.gif", 
                     "https://media.giphy.com/media/ZENe1zBJBDez4lGrXa/giphy.gif", 
                     "https://media.giphy.com/media/hrFoUUZQI5MfmLzdFk/giphy.gif", 
                     "https://media.giphy.com/media/cODov554cuegpBT4RD/giphy.gif", 
                     "https://media.giphy.com/media/f7STERTa61fiR1YBmq/giphy.gif", 
                     "https://media.giphy.com/media/KD17BLVo4xEt2nU6Lr/giphy.gif", 
                     "https://media.giphy.com/media/fYAuFiGR4uEQ1TrPGv/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    kick(){
      const links = ["https://media.giphy.com/media/Kbx0r2ZeuzZZE4LoLF/giphy.gif", 
                     "https://media.giphy.com/media/cPNSEOXZ7MegNJ6q93/giphy.gif", 
                     "https://media.giphy.com/media/LR6mw61R6DzupMylDo/giphy.gif", 
                     "https://media.giphy.com/media/WUaNzdB5n4K92oa6HW/giphy.gif", 
                     "https://media.giphy.com/media/fXWJT2m3l9DPWyxmK8/giphy.gif", 
                     "https://media.giphy.com/media/TjMCyt2PmcWFp7Kwrr/giphy.gif", 
                     "https://media.giphy.com/media/efTlQK9lBXohswMIJh/giphy.gif", 
                     "https://media.giphy.com/media/Y0bhvIb5bvmHPYeYNn/giphy.gif", 
                     "https://media.giphy.com/media/S3hd54AUvQ4kRoIeua/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    yaoi(){
      const links = ["https://media.giphy.com/media/VIiOM4PGR7XLonq2ng/giphy.gif", 
                     "https://media.giphy.com/media/SY31MSmx6dvAzORrwo/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    yuri(){
       const links = ["https://media.giphy.com/media/QBRy9eZ6mmXJncfbn5/giphy.gif", 
                     "https://media.giphy.com/media/ZBnjQiysril6w6RLSQ/giphy.gif", 
                     "https://media.giphy.com/media/Y3YIhfkPrBIcMohg8A/giphy.gif", 
                     "https://media.giphy.com/media/LqICLZe3HyDqB22Woy/giphy.gif", 
                     "https://media.giphy.com/media/LR5GZAjel7japHp9Xy/giphy.gif", 
                     "https://media.giphy.com/media/WUmSDAxty2aI8YVr38/giphy.gif", 
                     "https://media.giphy.com/media/fwWzeFtQ0OpUQPeYxi/giphy.gif", 
                     "https://media.giphy.com/media/cLMJa2Yclyu0NwpfBk/giphy.gif", 
                     "https://media.giphy.com/media/Qx9rBItbVSnqHDXPIC/giphy.gif", 
                     "https://media.giphy.com/media/dzCpO9MY7eDK9HxcVr/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    fuckyou(){
      const links = ["https://media.giphy.com/media/Kct9UpIeJL6yxOwPF0/giphy.gif", 
                     "https://media.giphy.com/media/dvb1loMgABlpyLjuKF/giphy.gif", 
                     "https://media.giphy.com/media/lrVtXhYrTo7WKXvJOb/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    sleep(){
      const links = ["https://media.giphy.com/media/dZ99BYrZpbM1sydocJ/giphy.gif", 
                     "https://media.giphy.com/media/L3d5IstM6NjveIVFPX/giphy.gif", 
                     "https://media.giphy.com/media/LnnlN5sf471E1gTRvf/giphy.gif",
                     "https://media.giphy.com/media/SWzsnsdlEaQYOOqz72/giphy.gif",
                     "https://media.giphy.com/media/Q7uHX9Svr4C95jRUKX/giphy.gif",
                     "https://media.giphy.com/media/cPfjyZD5Ts6zft0Oil/giphy.gif",
                     "https://media.giphy.com/media/lSshGTM5xkeniExwfs/giphy.gif",
                     "https://media.giphy.com/media/KEeLDTMhnoXt9nzTWZ/giphy.gif",
                     "https://media.giphy.com/media/KFiP19juIA7FOaQo5E/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
    },
    dance(){
      const links = ["https://media.giphy.com/media/f8PR3YXNeSF9InRPRE/giphy.gif", 
                     "https://media.giphy.com/media/Q7LNaJQB2sUhA8zKke/giphy.gif", 
                     "https://media.giphy.com/media/gdZSBttJReZa6Gflkl/giphy.gif",
                     "https://media.giphy.com/media/dBUhGWg1LgeIvogodK/giphy.gif",
                     "https://media.giphy.com/media/ehJ0PwnSzcKIAicA2r/giphy.gif",
                     "https://media.giphy.com/media/TKKrHDCkJm0P9rcFwY/giphy.gif",
                     "https://media.giphy.com/media/U5JFeJqG4vJIBIhf2y/giphy.gif",
                     "https://media.giphy.com/media/ZZTPHJJ1uk2CfmkZoj/giphy.gif",
                     "https://media.giphy.com/media/VgBNCbB3jmUYrlwuBR/giphy.gif",
                     "https://media.giphy.com/media/ZaWZ6oMAWgS6psEIz2/giphy.gif",
                     "https://media.giphy.com/media/ghIRlMA35kJW3uDRoP/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
   },
    blush(){
      const links = ["https://media.giphy.com/media/KdqiV6cJjPt27RgbDd/giphy.gif", 
                     "https://media.giphy.com/media/H47rUx9F92kjjIesY4/giphy.gif", 
                     "https://media.giphy.com/media/hTHmspYZZyPzr22Sma/giphy.gif",
                     "https://media.giphy.com/media/h3zYn68FTZIaa0Neet/giphy.gif",
                     "https://media.giphy.com/media/RfMN5aADBrXeBb8qx1/giphy.gif",
                     "https://media.giphy.com/media/fXyirwCeSaddSqwlVw/giphy.gif",
                     "https://media.giphy.com/media/dCL3GzLcr7BsfJ56Y6/giphy.gif",
                     "https://media.giphy.com/media/d9a1PqRzkKbk7K9UG4/giphy.gif",
                     "https://media.giphy.com/media/Me7yjG0ek5x2jCCOOu/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
   },
    confused(){
      const links = ["https://media.giphy.com/media/H5C6ygfroY1MTY98oL/giphy.gif", 
                     "https://media.giphy.com/media/Qal0dCuSM4PaZRfWQp/giphy.gif", 
                     "https://media.giphy.com/media/LklLNE22bvD88l0MEY/giphy.gif",
                     "https://media.giphy.com/media/RheYGiBI3jInE2o48e/giphy.gif",
                     "https://media.giphy.com/media/fTndmnKEWWwqsLhscW/giphy.gif",
                     "https://media.giphy.com/media/d9ZFzcNtjG7mekcM72/giphy.gif",
                     "https://media.giphy.com/media/dyLWrQEGiiASZeROea/giphy.gif",
                     "https://media.giphy.com/media/Za31i7iow8etebxk2j/giphy.gif"]
      const urls = Math.floor((Math.random() * links.length))
      const result = links[urls]
      return result;
   }
}
